FRAuthenticator/FRAuthenticator/Model/Mechanism/OATH/OathQRCodeParser.swift

```diff
@@ -155,5 +155,11 @@ struct OathQRCodeParser {
         if self.issuer.count == 0 {
             self.issuer = self.label
         }
+        
+        if self.issuer == "" && self.label == "" {
```

Comment:
Instead of assigning empty values I would suggest making those properties optional if they can be empty. Unless, there is a business reason not to.

Reply:
Furthermore, in various places we have comparisons against static strings:
Ex: item.name == "algorithm"

I would suggest for safety and future proofing to move those static strings as part of an Enum or a "Values" static helper file

Reply:
@george-bafaloukas-forgerock I agree with all suggested changes. However, I would suggest handle those in a future release.